modules/utxo-lib/package.json

```diff
@@ -17,9 +18,10 @@
     "javascript"
   ],
   "scripts": {
+    "build": "tsc --project tsconfig.build.json",
     "coverage-report": "nyc report --reporter=lcov",
     "coverage-html": "nyc report --reporter=html",
-    "coverage": "BITGO_UTXO_LIB_TEST_EXPECTED_COUNT=3539 nyc --check-coverage --branches 90 --functions 90 mocha --recursive",
+    "coverage": "npm run build && BITGO_UTXO_LIB_TEST_EXPECTED_COUNT=3539 nyc --check-coverage --branches 89 --functions 90 mocha",
```

Comment:
the coverage was reduced for spurious reasons (typescript boilerplate?)

Reply:
Do we need to build first? I think mocha is configured to run the tests with `ts-node`, so isn't this just extra work?

Reply:
we need it because `nyc` uses the `main` entry point from `package.json` (and somehow maps it back even when run with `ts-node`)